src/droplet.F

```diff
@@ -694,13 +746,18 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
     if(timestats.ge.1) time_droplet=time_droplet+mytime()
 
 #endif
-
+!$acc end data ! (pdata_neighbr,msk)
 
     ! free up the memory for temporary variables
 
-    deallocate(holes_ind)
-    if(allocated(Depart_ind)) deallocate(Depart_ind)
-    if(allocated(backfill_ind)) deallocate(backfill_ind)
+     !$acc exit data delete(holes_ind)
+     deallocate(holes_ind)
+
+     !$acc exit data delete(Depart_ind)
+     if(allocated(Depart_ind)) deallocate(Depart_ind)
+
+     !$acc exit data delete(backfill_ind)
```

Comment:
should this line be moved inside the if block below?

Reply:
Good catch.  I will add it to the if block.